### PR TITLE
And So Dies ZAS

### PR DESCRIPTION
**Pull request by Fox-McCloud**
With a plaque of shame.

### CODE REVIEW

**Comment by tigercat2000** [comment]
:heart: :heartbeat: :heart:

File: code/game/machinery/doors/door.dm [hunk 1]
```diff
@@ -100,10 +100,8 @@
 	return !density
 
 /obj/machinery/door/proc/bumpopen(mob/user as mob)
-	if(operating)	return
-//	if(user.last_airflow > world.time) //Fakkit //remind me to figure out the linda equiv
```
Comment (tigercat2000) [inline code/game/machinery/doors/door.dm]: I never did find out the linda equivalent.

**Comment by PJB3005** [comment]
LINDA.png


Removing ZAS was a mistake.

File: code/game/objects/structures/signs.dm [hunk 2]
```diff
@@ -146,8 +146,8 @@
 	icon_state = "kiddieplaque"
 
 /obj/structure/sign/atmosplaque
-	name = "\improper FEA Atmospherics Division plaque"
```
Comment (Iamgoofball) [inline code/game/objects/structures/signs.dm]: I think the FEA plaque should stay and the ZAS plaque should be next to it.

Comment (Alex6511) [inline code/game/objects/structures/signs.dm]: Agreed.

**Pull request merged**